### PR TITLE
fixed orientation guidance

### PR DESCRIPTION
**Pull request by rmeertens**
Line 612 was wrong ;)

### CODE REVIEW

File: sw/airborne/firmwares/rotorcraft/autopilot.c [hunk 1]
```diff
@@ -596,25 +596,27 @@ void autopilot_guided_update(uint8_t flags, float x, float y, float z, float yaw
...
-          x = stateGetPositionNed_f()->x + cosf(-psi) * x + sinf(-psi) * y;
-          y = stateGetPositionNed_f()->y - sinf(-psi) * x + cosf(-psi) * y;
+
+          setpoint.x = stateGetPositionNed_f()->x + cosf(-psi) * x + sinf(-psi) * y;
+          setpoint.y = stateGetPositionNed_f()->y - sinf(-psi) * x + cosf(-psi) * y;
         } else {                     // set position as offset in NED frame
-          x += stateGetPositionNed_f()->x;
-          y += stateGetPositionNed_f()->y;
+          setpoint.x += stateGetPositionNed_f()->x;
+          setpoint.x += stateGetPositionNed_f()->y;
```
Comment (flixr) [inline sw/airborne/firmwares/rotorcraft/autopilot.c]: should probably be `setpoint.y`?

**Comment by rmeertens** [comment]
Thanks Felix! You were right!

**Comment by flixr** [comment]
for future reference: this is a fix for a mistake in #1694

**Comment by kirkscheper** [comment]
Yea, dumb mistake on my part.

Kirk

**Comment by hooperfly** [comment]
Tested using the FRC. It works as expected. Thanks.

**Pull request merged**